pyproject.toml

```diff
@@ -62,4 +51,53 @@ show_column_numbers = true
 mypy_path = "../.stubs"
 # Exclude generated files
 # TODO Remove when we drop python 2
-exclude = '.*/config_models/.*\.py$'
\ No newline at end of file
+exclude = '.*/config_models/.*\.py$'
+
+[tool.ruff]
+exclude = [
+    ".eggs",
+    ".tox",
+    "build",
+    "compat.py",
+    "__init__.py",
+    "**/datadog_checks/dev/tooling/templates/*",
+    "**/datadog_checks/*/vendor/*",
+]
+target-version = "py37"
+line-length = 120
+select = [
```

Comment:
Same comment as for [this line](https://github.com/DataDog/integrations-core/pull/14212/files#diff-9b0e615a22b554712f68a1f287146b85ef3ae54f88e4b034706de513ed3c491aR71):

1. consider using `extend-select`
2. comment why we select this particular set of feedback codes.